plugins/dev-tools/src/addGUIControls.js

```diff
@@ -189,29 +198,39 @@ export function addGUIControls(createWorkspace, defaultOptions, config = {}) {
 
   // Basic options.
   const basicFolder = optionsFolder.addFolder('Basic');
+  //my-code
+  addTooltip(basicFolder,'Basic options like sound, comment etc...');
   populateBasicOptions(basicFolder, options, guiState, onChange);
 
   // Move options.
   const moveFolder = optionsFolder.addFolder('Move');
+  //my-code
+  addTooltip(moveFolder,'Move options like scrollbars, drag etc...');
```

Comment:
Why not set the tooltip of each of the actions in the move folder?

Reply:
i have to add tooltip to Zoom folder and , Grid folder as well

Reply:
added tooltip to move folder